### PR TITLE
[Merged by Bors] - feat: `comap` of a finite measure along a measurable equiv is finite

### PR DESCRIPTION
**Pull request by YaelDillies**
From PFR

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean [hunk 1]
```diff
@@ -1029,6 +1029,10 @@ instance [NeZero μ] : NeZero (μ univ) := ⟨measure_univ_ne_zero.2 <| NeZero.n
 theorem measure_univ_pos : 0 < μ univ ↔ μ ≠ 0 :=
   pos_iff_ne_zero.trans measure_univ_ne_zero
 
+lemma nonempty_of_neZero (μ : Measure α) [NeZero μ] : Nonempty α :=
+  have : μ Set.univ ≠ 0 := by simpa only [ne_eq, measure_univ_eq_zero] using NeZero.ne μ
```
Comment (b-mehta) [inline Mathlib/MeasureTheory/Measure/MeasureSpace.lean:1033]:
```suggestion
  have : μ univ ≠ 0 := NeZero.ne (μ univ)
```
looks like it should work here, though I haven't tested.

Comment (YaelDillies) [reply Mathlib/MeasureTheory/Measure/MeasureSpace.lean]: Nicely spotted

File: Mathlib/MeasureTheory/Measure/Typeclasses.lean [hunk 2]
```diff
@@ -122,6 +122,9 @@ theorem Measure.isFiniteMeasure_map {m : MeasurableSpace α} (μ : Measure α) [
   · rw [map_of_not_aemeasurable hf]
     exact MeasureTheory.isFiniteMeasureZero
 
+instance IsFiniteMeasure_comap_equiv (f : β ≃ᵐ α) [IsFiniteMeasure μ] :
```
[the remark this answers is not in the document]
Comment (YaelDillies) [reply Mathlib/MeasureTheory/Measure/Typeclasses.lean:125]: Brain was off :)

**Comment by kex-y** [comment]
Thanks! 
maintainer merge

**Comment by RemyDegenne** [comment]
Thanks!
bors r+

**Pull request closed without merging**